src/main/java/com/hotels/beeju/BeejuJUnitRule.java

```diff
@@ -49,10 +51,16 @@
   private final String driverClassName;
   private File metastoreLocation;
 
-  public BeejuJUnitRule(String databaseName) {
+  public BeejuJUnitRule(String databaseName, Map<String, String> configuration) {
```

Comment:
Any particular reason you chose a Map for the config rather than allowing the user to pass a HiveConf object?